kazoo/handlers/eventlet.py

```diff
@@ -13,6 +13,7 @@
 from eventlet.green import threading as green_threading
 from eventlet import queue as green_queue
 
+from kazoo.exceptions import KazooTimeoutError, TimeoutError
```

Comment:
I think that's fine, as it keep's the API consistent then. You can add ``# noqa`` at the end of the line and it'll pass the flake8.